scripts/ci/pylint_todo.txt

```diff
@@ -462,8 +462,8 @@
 ./tests/lineage/backend/test_atlas.py
 ./tests/lineage/test_lineage.py
 ./tests/macros/test_hive.py
-./tests/minikube/test_kubernetes_executor.py
-./tests/minikube/test_kubernetes_pod_operator.py
+./tests/integration/kubernetes/test_kubernetes_executor.py
```

Comment:
I fixed the issues _and_ also moved the files. These tests are actually `integration` tests, so I think we should start putting these kinds of tests in an integration folder to make it self-evident.

Reply:
One more thing - you can now also remove it it from pylint_todo_txt (if all pylint issues are fixed) - this way it will be checked during pre-commits/travis build.